Update Thanos to Mimir migration guide with relabelling tip (#11584)
grafana/mimir: docs/sources/mimir/set-up/migrate/migrate-from-thanos-or-prometheus.md

   You can get the external labels in the `meta.json` file of each block from the CSV file that's imported, and build the rewrite configuration accordingly.
   {{< /admonition >}}

   {{< admonition type="tip" >}}

>    Be sure to add `__tenant_id__` label with value of your target tenant. This is important for Mimir to successfully query the migrated data when tenant federation is enabled and Grafana data source is configured to query multiple tenants.
>    {{< /admonition >}}

Reviewer: ```suggestion
   Add the `__tenant_id__` label with the value of your target tenant. This is important for Mimir to successfully query the migrated data when you enable tenant federation and configure the Grafana data source to query multiple tenants.
   {{< /admonition >}}
```
